<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Auswertung</class>
 <widget class="QDialog" name="Auswertung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>679</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Auswertung -Wochenplan</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>250</height>
      </size>
     </property>
     <property name="title">
      <string>Austwerung</string>
     </property>
     <widget class="QCheckBox" name="ch_teilnehmer">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Teilnehmer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_ausbilder">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>40</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_suchen">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Suchen</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_unterrichtsthema">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>90</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>39</y>
        <width>87</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Datum von:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>82</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Datum bis:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="de_bis">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>109</width>
        <height>27</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2021</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>yyyy.MM.dd</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDateEdit" name="de_von">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>109</width>
        <height>27</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2021</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>yyyy.MM.dd</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>87</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ausbilder:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>90</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Unterrichtsthema:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ch_bewertung">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Bewertung</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tw_Table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
